src/patternfly/components/ActionList/action-list.scss

```
@@ -1,29 +1,38 @@
 // @debug $action-list; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$action-list} {
-  --#{$action-list}--m-icon--spacer: 0;
-  --#{$action-list}--child--spacer-base: var(--#{$pf-global}--spacer--md);
-  --#{$action-list}--group--spacer-base: var(--#{$pf-global}--spacer--2xl);
+:where(:root),
+:where(.#{$action-list}) {
+  --#{$action-list}--RowGap: var(--pf-t--global--spacer--sm);
+  --#{$action-list}--ColumnGap: var(--pf-t--global--spacer--sm);
+  --#{$action-list}--Padding: var(--pf-t--global--spacer--sm);
+  --#{$action-list}--inset: var(--#{$action-list}--Padding);
+  --#{$action-list}--LineHeight: var(--pf-t--global--font--line-height--body);
+  --#{$action-list}--FontSize: var(--pf-t--global--font--size--body--md);
+  --#{$action-list}--BackgroundColor: var(--pf-t--global--background--color--primary--default);
+
+  // * Action list icons
+  --#{$action-list}--m-icons--ColumnGap: 0;
 }
 
+// - Action list - Action list group
 .#{$action-list},
 .#{$action-list}__group {
-  --#{$action-list}--child--spacer: var(--#{$action-list}--child--spacer-base);
-  --#{$action-list}--group--spacer: var(--#{$action-list}--group--spacer-base);
-
   display: flex;
   align-items: center;
+}
 
-  > * + * {
-    margin-inline-start: var(--#{$action-list}--child--spacer);
-  }
-
-  > * + .#{$action-list}__group,
-  .#{$action-list}__group + * {
-    margin-inline-start: var(--#{$action-list}--group--spacer);
-  }
+// - Action list
+.#{$action-list} {
+  row-gap: var(--#{$action-list}--RowGap);
+  column-gap: var(--#{$action-list}--ColumnGap);
 
   &.pf-m-icons {
-    --#{$action-list}--child--spacer: var(--#{$action-list}--m-icon--spacer);
+    column-gap: var(--#{$action-list}--m-icons--ColumnGap);
```

Should this instead be:

```suggestion
    --#{$action-list}--ColumnGap: var(--#{$action-list}--m-icons--ColumnGap);
```

---

In past iterations of modifier implementation, updating variable vs property, especially where inheritance could be a potential issue, can result in unwanted results. Since this is a layout variable, seems best to update the property rather than var.